workshop/app/templates/application.hbs

```diff
@@ -1,3 +1,9 @@
-<h2 id="title">Welcome to Ember.js</h2>
+<div class="jumbotron">
+  <div class="container">
+    <h1>Bernice's Blog</h1>
```

Comment:
Lol no I don't care :) I just thought it was random and that would be correct it seems